synfig-studio/src/gui/dialogs/vectorizersettings.cpp

```diff
@@ -261,18 +255,6 @@ VectorizerSettings::on_comboboxtext_mode_changed()
 	}
 }
 
-void
-VectorizerSettings::on_finished()
```

Comment:
Please also remove declaration of this function from `synfig-studio/src/gui/dialogs/vectorizersettings.h`.

Reply:
done